plugins/module_utils/podman/podman_pod_lib.py

```diff
@@ -574,9 +573,14 @@ def running(self):
         """Return True if pod is running now."""
         if 'status' in self.info['State']:
             return self.info['State']['status'] == 'Running'
-        if 'status' in self.ps_info:
-            return self.ps_info['status'] == 'Running'
-        return self.info['State'] == 'Running'
+        elif isinstance(self.info['State'],str):
```

Comment:
1. `elif` is not necessary because you have `return` in `if` before, so better to keep it `if` as it was in previous commit
2. Need a space between `['State'],` and `str`, linters will complain.
And I'm not sure we need to change the previous behavior here.